spec/classes/puppet_config_spec.rb

```diff
@@ -130,20 +130,20 @@
         it 'should contain puppet.conf [main] with autosign = true' do
           verify_concat_fragment_contents(catalogue, 'puppet.conf+10-main', [
             '[main]',
-            '    autosign       = true',
+            '    autosign = true',
           ])
         end
       end
 
-      describe 'when autosign => $confdir/custom_autosign {mode = 664}' do
+      describe 'when autosign => /somedir/custom_autosign, autosign_mode => 664' do
```

Comment:
added to/puppet_server_config_spec.rb - a better test will be probably possible after #375, as there will be an explicit paramter also to the server class.